hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/OzoneClientUtils.java

```diff
@@ -205,8 +207,19 @@ public static FileChecksum getFileChecksumWithCombineMode(OzoneVolume volume,
     if (keyName.length() == 0) {
       return null;
     }
-    BaseFileChecksumHelper helper = new ReplicatedFileChecksumHelper(
-        volume, bucket, keyName, length, combineMode, rpcClient);
+    OmKeyArgs keyArgs = new OmKeyArgs.Builder().setVolumeName(volume.getName())
+        .setBucketName(bucket.getName()).setKeyName(keyName)
+        .setRefreshPipeline(true).setSortDatanodesInPipeline(true)
+        .setLatestVersionLocation(true).build();
+    OmKeyInfo keyInfo = rpcClient.getOzoneManagerClient().lookupKey(keyArgs);
+
+    if (keyInfo.getReplicationConfig()
```

Comment:
can you add a TODO comment here? Like:
```suggestion
    // TODO: return null util ECFileChecksum is implemented
    if (keyInfo.getReplicationConfig()
```

Reply:
Sure, I thought I added a comment somewhere. Let me do that right away.